### ISSUE TITLE
Add the fixed width feature to font awesome icons

### ISSUE THREAD

**Issue by Jinjinov** [Megabit/Blazorise#3992]
**Is your feature request related to a problem? Please describe.**
Font awesome icons have different widths. This doesn't look good when using different icons in rows and columns, because they don't align.

**Describe the solution you'd like**
Add `FixedWidth` to the `Icon` class, like the existing font awesome css class `fa-fw`

### Additional context
    .fa-fw {
        text-align: center;
        width: 1.25em;
    }

**Comment by stsrki** [comment]
How would this work with different `IconSize` values? They can range from `xs` to `lg`, and from `2em` to `10em`. From what I can see `fa-fw` is fixed to `1.25em`.

**Comment by Jinjinov** [comment]
Good point... right now I am using `Class="fa-fw"` and it works ok - I have eliminated the resizing when I switch from `Name="IconName.Edit"` to `Name="IconName.Save"`.

Could this be done in any other way that would take all the different sizes into consideration?

All I want is that my UI doesn't resize when I hide the edit icon and show the save icon in its place.
